src/lib/gql.ts

```diff
@@ -55,7 +55,7 @@ export const createApolloClient = (config: Config) => {
 	const authLink = new ApolloLink((operation, forward) => {
 		operation.setContext(({ headers }: { headers: any }) => ({
 			headers: {
-				authorization: `Bearer ${jwt}`,
+				authorization: jwt ? `Bearer ${jwt}` : '',
```

Comment:
Perhaps we shouldnt be setting an authorization header if we have no token?

Reply:
Sure

```
	const authHeader = jwt ? {authorization: `Bearer ${jwt}`} : {};	

	const authLink = new ApolloLink((operation, forward) => {
		operation.setContext(({ headers }: { headers: any }) => ({
			headers: {
				...authHeader,
				...headers,
			},
```

Reply:
Not asking for change: this has lower cognitive burden.
```
const authHeader = jwt && {authorization: `Bearer ${jwt}\`};
```

Reply:
Doesn't that break the next section's `...authHeader` spread?

Reply:
nup. try it out ;) 

`{ ...false } == { }`

Reply:
Weird!